release_templates/capi/capo/<=v0.5.0/cluster_config.yaml

```diff
@@ -0,0 +1,40 @@
+bastion:
```

Comment:
This base is completely unused. We do not have any clusters below 0.5.0 anywhere anymore.